lib/routes/account.js

```diff
@@ -81,19 +80,22 @@ module.exports = (log, db, mailer, Password, config, customs, signinUtils, push)
           flowBeginTime = request.payload.metricsContext.flowBeginTime
         }
 
-        customs.check(request, email, 'accountCreate')
-          .then(deleteAccountIfUnverified)
-          .then(setMetricsFlowCompleteSignal)
-          .then(generateRandomValues)
-          .then(createPassword)
-          .then(createAccount)
-          .then(createSessionToken)
-          .then(sendVerifyCode)
-          .then(createKeyFetchToken)
-          .then(recordSecurityEvent)
-          .then(createResponse)
-          .then(reply, reply)
-
+       try {
+           await customs.check(request, email, 'accountCreate')
+           await deleteAccountIfUnverified()
+           await setMetricsFlowCompleteSignal()
+           await generateRandomValues()
+           await createPassword()
+           await createAccount()
+           await createSessionToken()
+           await sendVerifyCode()
+           await createKeyFetchToken()
+           await recordSecurityEvent()
+           const resp = await createResponse()
```

Comment:
@deeptibaghel could you check this here, I don't think there is anything async in that function, can we drop `await` here and just return?